### PR TITLE
Log Control

### PR DESCRIPTION
**Pull request by Saiby100**

### CODE REVIEW

File: packages/api/src/routes/endpoints/control/controlLog.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import { type } from '@lowdefy/helpers';
+
+async function controlLog(context, routineContext, { control }) {
+  const { logger, evaluateOperators } = context;
+  const { items } = routineContext;
+
+  logger.debug({ event: 'debug_control_log' });
+  const log = evaluateOperators({ input: control[':log'], items, location: 'TODO:' });
+  const logLevel = control[':level'] ?? 'info';
```
Comment (SamTolmay) [inline packages/api/src/routes/endpoints/control/controlLog.js:25]: Should we evaluate operators for the log level? I'm not sure?

Comment (Saiby100) [reply packages/api/src/routes/endpoints/control/controlLog.js:25]: I don't see why not

Comment (Saiby100) [reply packages/api/src/routes/endpoints/control/controlLog.js]: Updated in a355bb3cddb1ea91c8c4b6fd8a45f2af41c4dff7

Comment (SamTolmay) [reply packages/api/src/routes/endpoints/control/controlLog.js]: Please add test case that operators are evaluated for log level

Comment (Saiby100) [reply packages/api/src/routes/endpoints/control/controlLog.js]: Added in a355bb3cddb1ea91c8c4b6fd8a45f2af41c4dff7

**Review by SamTolmay — APPROVED** [review]

File: packages/api/src/routes/endpoints/control/controlLog.js (hunk 1 above)
Comment (SamTolmay) [reply packages/api/src/routes/endpoints/control/controlLog.js]: thanks

**Pull request merged**